LabExT/Movement/Calibration.py

```diff
@@ -552,6 +571,80 @@ def move_absolute(self,
             z=stage_coordinate.z,
             wait_for_stopping=wait_for_stopping)
 
+    @assert_minimum_state_for_coordinate_system(
+        chip_coordinate_system=State.SINGLE_POINT_FIXED)
```

Comment:
Do I understand this correctly that this fct disallows using `lift_stage_absolute` in stage coordinates?

Reply:
Yep, because we need the position in chip coordinates. 
I am working on a `lift_stage_relative` method (PR is almost ready) that moves the stages up and down by just adding an offset. Here we need at least the state "Coordinate System fixed"
The idea was that we lift fixed relative for a single point (i.e., take the `lift_stage_relative` method) and take the `lift_stage_absolute` method from Kabsch (since the tilting is then in).

Reply:
#104